include/circt/Tools/circt-lec/Passes.td

```diff
@@ -0,0 +1,53 @@
+//===-- Passes.td - LEC Transforms pass definition file ----*- tablegen -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This file contains definitions for LEC transformation passes.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef CIRCT_TOOLS_CIRCT_LEC_PASSES_TD
+#define CIRCT_TOOLS_CIRCT_LEC_PASSES_TD
+
+include "mlir/Pass/PassBase.td"
+include "mlir/Rewrite/PassUtil.td"
+
+def ConstructLEC : Pass<"construct-lec", "::mlir::ModuleOp"> {
+  let summary = "Lower CIRCTs core dialects to a LEC problem statement";
+  let description = [{
+    Lowers HW and Comb operations to the SMT and Func dialect. HW modules become
+    functions that take SMT bit-vectors as input and return SMT bit-vectors
+    according to the original ports of the module. The Comb operations in the
```

Comment:
It doesn't look from the tests like the Comb ops are converted to SMT by this pass?

Reply:
Put this in an unclear place - it's addressing the sentence that runs off the end of the block GitHub previews!

Reply:
Oh, I should have noticed that. It's just the version from the experimental branch without taking the redesign into consideration. Thanks!